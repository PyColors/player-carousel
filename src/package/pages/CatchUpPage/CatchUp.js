import React from "react";

const CatchUp = () => (
    <h3>
        Catch Up Page
    </h3>
);
export default CatchUp;
