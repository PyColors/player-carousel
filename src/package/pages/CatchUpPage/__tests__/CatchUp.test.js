import React from "react";
import CatchUp from "../CatchUp";

it("Should renders CatchUp page without exploding", () => {
    const wrapper = shallow(<CatchUp>Hello Jest!</CatchUp>);
    expect(wrapper).toMatchSnapshot();
    expect(wrapper).toHaveLength(1)
});
