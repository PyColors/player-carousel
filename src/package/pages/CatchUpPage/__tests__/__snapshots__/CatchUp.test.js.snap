// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should renders CatchUp page without exploding 1`] = `
<h3>
  Catch Up Page
</h3>
`;
